ambition_utils/rrule/tests/model_tests.py

```diff
@@ -410,6 +411,59 @@ def test_get_next_occurrence_force(self):
         # Coverage for returning early
         self.assertEqual(rule.update_next_occurrence(), None)
 
+    def test_get_next_occurrence_dst(self):
+        """
+        What happens across DST changes?
+        """
+
+        # Create the params to create the rule
+        params = {
+            'freq': rrule.DAILY,
+            'interval': 1,
+            'dtstart': datetime.datetime(2022, 10, 29),
+            'until': datetime.datetime(2022, 11, 1),
+            'byhour': 10,
+        }
+        
+        timezone = pytz.timezone('Europe/Kiev')
+        format = '%Y-%m-%d %H:%M'
+
+        # Create the rule
+        rule = RRule.objects.create(
+            rrule_params=params,
+            occurrence_handler_path='ambition_utils.rrule.tests.model_tests.MockHandler',
+            time_zone=timezone,
+        )
+
+        # Assert the initial values
+        # Europe/Kiev is UTC + 3 prior to 10/30/22. (10 meeting is 7 UTC.)
```

Comment:
Should this comment be `(10 Europe/Kiev is 7 UTC)` instead of `(10 meeting is 7 UTC.)`?

Reply:
That's much clearer. Thanks for calling it out.